defaults/01_redhat_operators.cr.yaml

```diff
@@ -11,6 +11,8 @@ spec:
   displayName: "Red Hat Operators"
   publisher: "Red Hat"
   priority: -100
+  grpcPodConfig:
+    securityContextConfig: legacy
```

Comment:
why are we setting this to legacy, and not restricted?  These catalogs should be able to run restricted, no?

Reply:
Yikes, yeah they should be set to restricted, great catch.